custom_components/google_home/models.py

```diff
@@ -84,6 +85,14 @@ def get_next_timer(self) -> Optional[GoogleHomeTimer]:
         timers = self.get_sorted_timers()
         return timers[0] if timers else None
 
+    def set_do_not_disturb_status(self, status: str) -> None:
```

Comment:
Why `str`?
```suggestion
    def set_do_not_disturb_status(self, status: bool) -> None:
```

Reply:
Because `mypy` complains about casting a `str` to a bool when we get it from the endpoint in `api.py`. But you are right it should preferably be a bool. 😄

Reply:
I think it should be caster in `api.py`. Also be aware that:
```python
>>> bool("false")
True
>>> bool("False")
True
```

Reply:
I moved it to `api.py` 😄 Sometimes the errors are hard to understand, but it works now.